### ISSUE TITLE
Valine的阅读量数字无法在首页显示

### ISSUE THREAD

**Issue by time-todie** [SukkaW/hexo-theme-suka#131]
**我确认我已经查看了** (选择时请使用 "x" 取代方括号中的空格)

- [x ] [Hexo documentation | Hexo 文档](https://hexo.io/docs/)
- [x] [Suka Theme documentation | Suka Theme 文档](https://theme-suka.skk.moe/docs/)
- [x] [Suka theme issues | Suka 主题的其它 issue](https://github.com/SukkaW/hexo-theme-suka/issues?utf8=%E2%9C%93&q=is%3Aissue)

----

### Node.js 和 NPM 的相关信息

node v10.16.3
npm -v6.9.0

### 依赖包相关信息

站点目录下的package.json：
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "build": "hexo generate",
    "clean": "hexo clean",
    "deploy": "hexo deploy",
    "server": "hexo server"
  },
  "hexo": {
    "version": "4.2.0"
  },
  "dependencies": {
    "hexo": "^4.2.0",
    "hexo-blog-encrypt": "^3.0.12",
    "hexo-deployer-git": "^2.1.0",
    "hexo-generator-archive": "^1.0.0",
    "hexo-generator-category": "^1.0.0",
    "hexo-generator-feed": "^2.2.0",
    "hexo-generator-index": "^1.0.0",
    "hexo-generator-tag": "^1.0.0",
    "hexo-hide-posts": "^0.1.0",
    "hexo-renderer-ejs": "^1.0.0",
    "hexo-renderer-marked": "^2.0.0",
    "hexo-renderer-stylus": "^1.1.0",
    "hexo-server": "^1.0.0"
  }
}

主题目录下package.json：
  "name": "hexo-theme-suka",
  "version": "1.3.3",
  "description": "Modern, powerful and simple theme for Hexo",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "setup": "npm cache clean && npm install",
    "lint": "bash lint.sh"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/SukkaW/hexo-theme-suka.git"
  },
  "keywords": [
    "hexo",
    "hexo-theme",
    "suka",
    "sukkaw"
  ],
  "author": "SukkaW",
  "license": "GPL-3.0",
  "bugs": {
    "url": "https://github.com/SukkaW/hexo-theme-suka/issues"
  },
  "homepage": "https://github.com/SukkaW/hexo-theme-suka#readme",
  "devDependencies": {
    "ejs-lint": "0.3.0",
    "eslint": "6.5.1",
    "gulp": "4.0.2",
    "gulp-autoprefixer": "7.0.1",
    "gulp-clean-css": "4.2.0",
    "gulp-rename": "1.4.0",
    "gulp-uglify": "3.0.2",
    "sass-lint": "1.13.1"
  },
  "dependencies": {
    "node-prismjs": "0.1.2",
    "prismjs": "1.17.1",
    "qr-image": "3.2.0"
  }
}

### Hexo 版本号

INFO  Checking dependencies
INFO  Loading Suka Theme Plugins
hexo: 4.2.0
hexo-cli: 3.1.0
os: Windows_NT 10.0.17763 win32 x64
http_parser: 2.8.0
node: 10.16.3
v8: 6.8.275.32-node.54
uv: 1.28.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 64
nghttp2: 1.39.2
napi: 4
openssl: 1.1.1c
icu: 64.2
unicode: 12.1
cldr: 35.1
tz: 2019a

### 站点配置文件和主题配置文件

全部站点配置文件：
# Hexo Configuration
## Source: https://github.com/hexojs/hexo/

# Site
title: ****
subtitle: ''
description: ''
keywords: 
author: *****
language: zh-CN
timezone: ''

# URL
## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
url: https://********.com
root: /
permalink: :year/:month/:day/:title/
permalink_defaults:
pretty_urls:
  trailing_index: true # Set to false to remove trailing 'index.html' from permalinks
  trailing_html: true # Set to false to remove trailing '.html' from permalinks

# Directory
source_dir: source
public_dir: public
tag_dir: tags
archive_dir: archives
category_dir: categories
code_dir: downloads/code
i18n_dir: :lang
skip_render:

# Writing
new_post_name: :title.md # File name of new posts
default_layout: post
titlecase: false # Transform title into titlecase
external_link:
  enable: true # Open external links in new tab
  field: site # Apply to the whole site
  exclude: ''
filename_case: 0
render_drafts: false
post_asset_folder: false
relative_link: false
future: true
highlight:
  enable: true
  line_number: true
  auto_detect: false
  tab_replace: ''
  wrap: true
  hljs: false

# Home page setting
# path: Root path for your blogs index page. (default = '')
# per_page: Posts displayed per page. (0 = disable pagination)
# order_by: Posts order. (Order by date descending by default)
index_generator:
  path: ''
  per_page: 10
  order_by: -date

# Category & Tag
default_category: uncategorized
category_map:
tag_map:

# Metadata elements
## https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta
meta_generator: true

# Date / Time format
## http://momentjs.com/docs/#/displaying/format/
date_format: YYYY-MM-DD
time_format: HH:mm:ss
## Use post's date for updated date unless set in front-matter
use_date_for_updated: false

# Pagination
## Set per_page to 0 to disable pagination
per_page: 10
pagination_dir: page

# Include / Exclude file(s)
## include:/exclude: options only apply to the 'source/' folder
include:
exclude:
ignore:

# Extensions
## Themes: https://hexo.io/themes/
theme: suka 

# Deployment
## Docs: https://hexo.io/docs/deployment.html
deploy:
    type: git
    repo: git@github.com:*****/*****.github.io.git
    branch: master

# Suka Theme config
# Documents: https://theme-suka.skk.moe/docs/
suka_theme:
  search:
    path: search.json
    field: post # Page | Post | All. Default post
  prism:
    line_number: true
    theme: default

feed:
  type: atom
  path: atom.xml
  limit: 20
  hub:
  content:
  content_limit: 140
  content_limit_delim: ' '
  order_by: -date
  icon: icon.png
  autodiscovery: true
  template:

 # hexo-hide-posts
hide_posts:
  # 可以改成其他你喜欢的名字
  filter: hide
  # 指定你想要传递隐藏文章的 generator，比如让所有隐藏文章在存档页面可见
  # 常见的 generators 有：index, tag, category, archive, sitemap, feed, etc.
  public_generators: []
  # 为隐藏的文章添加 noindex meta 标签，阻止搜索引擎收录
  noindex: true 

部分主题配置文件：
comment: 
  use: valine
  disqus:
    shortname: # disqus shortname
  disqusjs:
    shortname:
    siteName:
    api:
    apikey:
    admin:
    adminLabel:
  changyan:
    appid:
    conf:
    thread_key_type: path # id | path. Only when you generate static id for each post before you using 'id'
  livere:
    data_uid:
  gitment:
    repo:
    owner:
    client_id:
    client_secret:
  gitalk:
    repo:
    owner:
    client_id:
    client_secret:
  valine:
    leancloud_appId:   # leancloud application app id
    leancloud_appKey:   # leancloud application app key
    notify: false # valine mail notify (true/false) https://github.com/xCss/Valine/wiki
    verify: false # valine verify code (true/false)
    pageSize: 10 # comment list page size
    avatar: identicon # gravatar style https://valine.js.org/#/avatar
    lang: zh-cn # i18n
    placeholder: ************** # valine comment input placeholder(like: Please leave your footprints )
    guest_info: nick,mail,link #valine comment header info
  wildfire:
    database_provider: firebase # firebase or wilddog
    wilddog_site_id:
    firebase_api_key:
    firebase_auth_domain:
    firebase_database_url:
    firebase_project_id:
    firebase_storage_bucket:
    firebase_messaging_sender_id:
    theme: light # light or dark
    locale: en # en or zh-CN
  facebook:
    colorschme: light # light | dark
    numposts: 10
    orderby: social # social | reverse_time | time
    # You can only fill in one of them below when configuring.
    # If you fill in both of them then appid method will be used.
    # Read facebook developer documents for more info:
    # https://developers.facebook.com/docs/plugins/comments/#moderation-setup-instructions
    admin_fb_appid:
    admin_fb_username: # Currently "Suka" don't support more than one admin. You can open a new issue if you really need one.

# Site Analytics
# Suka theme has a built in Baidu's, Google's website analytics and CNZZ's analytics service
analytics:
  google_site_id: UA-150232869-1
  gtags_site_id:
  baidu_site_id:
  cnzz_site_id:
  tencent_site_id:
  tencent_mta_id:

# Views Analytics
# Busuanzi
# The counter by https://busuanzi.ibruce.info
busuanzi:
  enable: false
  # Site views analytics support either pv or uv
  # will be show at footer
  site_uv:
    enable: true
    before: # html tag support
    after: Viewers # html tag support
    divider: '|' # html tag supported, something like '|' ',' '·' even '<br>', if you dont want any divider simply leave it blank,
    offset: 0
  site_pv:
    enable: false
    before: # html tag support
    after: Views # html tag support
    divider: # html tag supported, something like '|' ',' '·' even '<br>', if you dont want any divider simply leave it blank,
    offset: 0
  # Post views analytics
  # will be show at post header info
  post_pv:
    enable: true
    before: # html tag support
    after: Views # html tag support

# The counter by https://valine.js.org/visitor.html
valine_counter:
  # ATTENTION! You should use valine comment system to enable this feature!
  enable: true
  # By use config.url as id to count all site pv
  site_pv:
    enable: false
    before: # html tag supported
    after: # html tag supported
    divider: # html tag supported, something like '|' ',' '·' even '<br>', if you dont want any divider simply leave it blank,
  # Show pv for each post on index (Exprimental)
  index_post_pv:
    enable: true
    before: 阅读： # html tag supported
    after: # html tag supported
  # Show pv on post page
  post_pv:
    enable: true
    before: 阅读： # html tag supported
    after: # html tag supported

# ---------------------------------------------------------------
# Plugin Settings
# ---------------------------------------------------------------
# Search
# Available value of "use":
#     local | google | swiftype
search:
  use: local
  # Swiftype
  swiftype_key:
### 报错日志

----

### 我使用的平台，如 Android、iOS、Windows、Linux、MacOS 等
Windows 10

### 我使用的浏览器（如  Chrome 58.0.3029.110 64 位）
chrome浏览器版本 79.0.3945.88

### 预期结果和你的实际结果
首页本该显示文章的阅读量数字，但是首页上所有阅读量数字全部为0。文章内的阅读量统计数字很正常。

----

### 和 issue 有关的在线演示页面，推荐使用 JSFiddle 和 gist
如图。
![1](https://user-images.githubusercontent.com/56446819/75768800-06d46000-5d80-11ea-8f76-87c4ed973634.PNG)
